pkgs/applications/virtualization/driver/win-spice/default.nix

```diff
@@ -2,50 +2,56 @@
 
 let
   src_usbdk_x86 = fetchurl {
-    url = "https://www.spice-space.org/download/windows/usbdk/UsbDk_1.0.4_x86.msi";
-    sha256 = "17hv8034wk1xqnanm5jxs4741nl7asps1fdz6lhnrpp6gvj6yg9y";
+    url = "https://www.spice-space.org/download/windows/UsbDk/UsbDk_1.0.22_x86.msi";
+    sha256 = "1vr8kv37wz6p3xhawyhwxv0g7y89igkvx30zwmyvlgnkv3h5i317";
   };
 
   src_usbdk_amd64 = fetchurl {
-    url = "https://www.spice-space.org/download/windows/usbdk/UsbDk_1.0.4_x64.msi";
-    sha256 = "0alcqsivp33pm8sy0lmkvq7m5yh6mmcmxdl39zjxjra67kw8r2sd";
+    url = "https://www.spice-space.org/download/windows/UsbDk/UsbDk_1.0.22_x64.msi";
+    sha256 = "19b64jv6pfimd54y0pphbs1xh25z41bbblz64ih6ag71w6azdxli";
   };
 
   src_qxlwddm = fetchurl {
-    url = "https://people.redhat.com/~vrozenfe/qxlwddm/qxlwddm-0.11.zip";
-    sha256 = "082zdpbh9i3bq2ds8g33rcbcw390jsm7cqf46rrlx02x8r03dm98";
+    url = "https://www.spice-space.org/download/windows/qxl-wddm-dod/qxl-wddm-dod-0.21/spice-qxl-wddm-dod-0.21.zip";
+    sha256 = "0yjq54gxw3lcfghsfs4fzwipa9sgx5b1sn3fss6r5dm7pdvjp20q";
```

Comment:
It wouldn't hurt if we would substitute those version numbers, too.

Reply:
Sure, especially as the qxlwddm version is reused as the package version.